www/_includes/termine_en.md

```diff
@@ -0,0 +1,39 @@
+{% for termin in page.termine %}
+<p {% if notfirst %}class="dim"{% endif %}>
+	{% assign notfirst = true %}
+
+	<b>date (yyyy-mm-dd): {{ termin.date | escape }}</b> from 19h<br>
+	{% if termin.override != "" %}
+		{{ termin.override | escape }}
+	{% elsif termin.stammtisch %}
+		<a href="stammtisch.html">Restaurant (“Stammtisch”)</a><br/>
+		<a href="yarpnarp.html">please RSVP</a><br/>
+		{% if termin.location != "" %}
+			{% for st in site.pages %}
+				{% unless st.layout == "stammtisch" %}
+					{% continue %}
+				{% endunless %}
+				{% unless st.locname == termin.location %}
+					{% continue %}
+				{% endunless %}
+				{% assign done = true %}
+				Location: <a href="{{ st.url }}">{{ termin.location | escape }}</a>
+			{% endfor %}
+			{% unless done %}
+				Location: <a href="stammtisch.html">{{ termin.location | escape }}</a>
+			{% endunless %}
+		{% else %}
+			Location: TBA
+		{% endif %}
+	{% else %}
+		regular meeting (<a href="anfahrt.html">Route?</a>)<br>
+		lightning talk:
```

Comment:
I don't agree with the characterization as "lightning talk" here. I'd say "short talk" or just "talk", but lightning talk, to me, implies that there will be a series of quick-fire talks with a specific format.